.gitignore

```diff
@@ -1 +1,14 @@
 node_modules/
+
```

Comment:
There's no need to include additional information in your `.gitignore`, it's not the README.md. I think these notes are better off in your comment on the Pull Request